app/core/vueRouter.js

```diff
@@ -7,9 +7,7 @@ import SchoolAdminTeacherView from 'app/views/school-administrator/dashboard/Sch
 import TeacherClassView from 'app/views/courses/TeacherClassView.vue'
 import TeacherStudentView from 'app/views/teachers/classes/TeacherStudentView.vue'
 
-Vue.use(VueRouter)
```

Comment:
Could this cause routing breakage!?

Reply:
Nope, this was an artifact from the original implementation that should have been removed.  This is called here https://github.com/codecombat/codecombat/blob/dfe2860fcb4768462cc38fa4f947a558dc2af4c7/app/core/initialize.coffee#L11

Removing this because the linter was yelling at me for an uninitialized global :)

Reply:
haha! Fantastic =D